src/js/components/Heading/StyledHeading.js

```diff
@@ -42,6 +42,20 @@ const sizeStyle = props => {
   return '';
 };
 
+const fontFamily = props => {
+  const levelStyle = props.theme.heading.level[props.level];
+  if (levelStyle && levelStyle.family) {
```

Comment:
Can we preserve the `font.family` pattern to be more consistent? As in:
```
heading: {
  level: {
    1: {
      font: {
        family: ...
```

Reply:
and done! ty!